src/renderer/src/routes/(MapTabs)/_Map.tsx

```diff
@@ -1,34 +1,99 @@
-import * as React from 'react'
-import { Paper } from '@mui/material'
+import { Suspense } from 'react'
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined'
+import PostAddIcon from '@mui/icons-material/PostAdd'
+import SettingsIcon from '@mui/icons-material/Settings'
+import { Box, CircularProgress, Paper } from '@mui/material'
 import Tab from '@mui/material/Tab'
 import Tabs from '@mui/material/Tabs'
+import { styled } from '@mui/material/styles'
 import { Outlet, createFileRoute, useNavigate } from '@tanstack/react-router'
+import { defineMessages, useIntl } from 'react-intl'
 
+import { VERY_LIGHT_GREY, WHITE } from '../../colors'
+import { Text } from '../../components/Text'
 import type { FileRoutesById } from '../../routeTree.gen'
 
+const m = defineMessages({
+	setting: {
+		id: 'tabBar.label.settings',
+		defaultMessage: 'Settings',
+	},
+	about: {
+		id: 'tabBar.label.about',
+		defaultMessage: 'About',
+	},
+})
+
+const Container = styled('div')({
+	display: 'flex',
+	backgroundColor: WHITE,
+	height: '100%',
+})
+
+const MapTabStyled = styled(MapTab)({
+	width: 60,
+})
+
 export const Route = createFileRoute('/(MapTabs)/_Map')({
 	component: MapLayout,
 })
 
 export function MapLayout() {
 	const navigate = useNavigate()
-	const renderCount = React.useRef(0)
-	renderCount.current = renderCount.current + 1
+	const { formatMessage } = useIntl()
 	return (
-		<div>
-			<Tabs
-				onChange={(_, value) => navigate({ to: value as MapTabRoute })}
-				orientation="vertical"
-			>
-				<MapTab label="Tab 1" value={'/tab1'} />
-				<MapTab label="Tab 2" value={'/tab2'} />
-			</Tabs>
-			<Paper>
-				<Outlet />
+		<Container>
+			<Paper elevation={3} sx={{ display: 'flex' }}>
+				<Tabs
+					sx={{ '& .MuiTabs-flexContainer': { height: '100%' } }}
+					onChange={(_, value) => navigate({ to: value as MapTabRoute })}
+					orientation="vertical"
+				>
+					{/* PostAddIcon is a placeholder icon */}
+					<MapTabStyled
+						data-testid="tab-observation"
+						icon={<PostAddIcon />}
+						value={'/tab1'}
+					/>
+					<Box flexGrow={1} />
+
+					<MapTabStyled
+						icon={<SettingsIcon />}
+						label={
+							<Text style={{ fontSize: 10 }} kind="title">
+								{formatMessage(m.setting)}
+							</Text>
+						}
+						value={'/tab2'}
+					/>
+					<MapTabStyled
+						icon={<InfoOutlinedIcon />}
+						label={
+							<Text style={{ fontSize: 10 }} kind="title">
+								{formatMessage(m.about)}
+							</Text>
+						}
+						value={'/tab2'}
```

Comment:
Having both settings and about map to tab2 is very confusing. Is this just temporary?

Reply:
Ok thanks